README.md

```diff
@@ -1,2 +1,16 @@
 # Tokenizer
 Library used by Meilisearch to tokenize queries and documents
+
+## Role
+
+The tokenizer’s role is to take a sentence or phrase and split it into smaller units of language, called tokens. It finds and retrieves all the words in a string based on the language’s particularities.  
+
+## Particularities
+
+MeiliSearch’s tokenizer is modular. It goes field by field, determining the most likely language for the field and running a different pipeline for each language.
+
+The tokenizer identifies words made of one or more Hanzi / Kanji characters. In addition, a single search query will give results in both traditional and simplified Chinese.
```

Comment:
```suggestion
Pipelines include language-specific processes. For example, the Chinese pipeline converts all text into simplified Chinese before tokenization, allowing a single search query to give results in both traditional and simplified Chinese.
```

Reply:
Sentence about hanzi/kanji characters does not feel relevant to me, so I cut it in my suggestion. If there were a section on the Chinese pipeline then it would make sense there, but doesn't really have to do w/ the tokenizer itself IMO.